src/NextSteps/index.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import React from 'react'
+import { Link as GatsbyLink } from 'gatsby'
+import { css } from '@emotion/core'
+import { Heading4 } from '../Heading'
+import { View } from '@react-spectrum/view'
+import '@spectrum-css/typography'
+import '@spectrum-css/link'
+import Document from '@spectrum-icons/workflow/Document'
+import PropTypes from 'prop-types'
+
+const NextSteps = ({ pages }) => {
+  return pages.length > 0 ? (
+    <View elementType='nav' marginTop='size-800'>
```

Comment:
If this `View` gets rendered as a `nav` do we need to apply an `aria-label` to it?

Reply:
I think the nav is not needed actually.